### ISSUE TITLE
Emulator Env Variable setting not persisting 

### ISSUE THREAD

**Issue by MTocchettoP** [googleapis/google-cloud-datastore#209]
Hi, I decided to open this issue as I couldn't find anything related to this problem. This might be a very specific problem so I hope if nothing else this will serve as documentation for future devs. 

I am running my emulator on a Windows Server R2 VM, the issue I was having is that my code was creating the entities on the actual datastore and not on the emulator, even though the emulator were running and the variables were being set (I thought). 

Ok, so the command to automatically set the env variables is 

`gcloud beta emulators datastore env-init > set_vars.cmd && set_vars.cmd`

What it does is:  `gcloud beta emulators datastore env-init` contain the actual commands like `set DATASTORE_EMULATOR_HOST=XXXX` , that information is then piped to a file called set_vars.cmd and then this file is ran.

The problem start when you decide to put this command on a BAT or CMD file, it simply doesn't work. I tried a lot of things, including running it as a PS1 file with unrestricted execution policy, to no avail.

Alright, I can live without automatically setting this in a bat file, let me manually run the commands in set_vars.cmd. At first it seems to work, the prompt say it was set, you call an ECHO and it show you the variable's value but when you close the command prompt and open it again, the variables are not there.

After some research, the way I found to make these sets persist are using SETX like `setx DATASTORE_EMULATOR_HOST localhost:8081` . Now, after you issue this command, ECHO won't show you anything on the command prompt, you have to close it an open another one and the variable will be there (This is not necessary to set the variable, just to see it is there). Alternatively you can go to the UI for setting env vars and you will see it there, with a small catch, SETX only set user variables, not System variables but this don't seem to interfere with the emulator usage.

Also, the cherry on top, because a single problem was no enough, if you are using IntelliJ you need to close it and open again after setting the variables, yes... restart the IDE (not sure if this also happens with other IDEs).

To sum up, assuming I am doing something wrong and it would be possible to set this variables through a BAT or CMD file, it would be useful if `gcloud beta emulators datastore env-init` had an option of outputting the commands using `SETX`

Thank you for your support.

**Comment by meredithslota** [comment]
https://cloud.google.com/datastore/docs/tools/datastore-emulator#windows includes updated documentation and functionality.

**Comment by meredithslota** [comment]
Hi there — thank you for filing this! I think this is a feature request for how emulators are supported in `gcloud`. I recommend following the support and feature request guidance here (https://cloud.google.com/sdk/docs/getting-support) if this is still an issue for you. Thanks!
